lib/MiLight/MiLightRadioFactory.cpp

```diff
@@ -6,7 +6,7 @@ MiLightRadioFactory* MiLightRadioFactory::fromSettings(const Settings& settings)
       return new NRF24Factory(settings.csnPin, settings.cePin);
 
     case LT8900:
-      return new LT8900Factory(settings.csnPin, settings.resetPin, settings.cePin);
+      return new LT8900Factory(settings.cePin, settings.resetPin, settings.csnPin);
```

Comment:
I apologize in advance for being dense here. Just wanna make sure I understand:

It's a little funky that this is the correct order when the arguments are passed in through this path:

```c++
  LT8900Factory(uint8_t csPin, uint8_t resetPin, uint8_t pktFlag);

...

MiLightRadio* LT8900Factory::create(const MiLightRadioConfig& config) {
  return new LT8900MiLightRadio(_csPin, _resetPin, _pktFlag, config);
}

...

    LT8900MiLightRadio(byte byCSPin, byte byResetPin, byte byPktFlag, const MiLightRadioConfig& config);
```

Shouldn't the way it is be the correct order? It's a little confusing that what's in the "ce" setting eventually gets passed into `byCSPin`. If you flipped the setting values for your cs/ce pins without this change, would you get the same result?